modules/social_features/social_event/social_event.module

```diff
@@ -561,7 +627,8 @@ function social_event_form_alter(&$form, FormStateInterface $form_state, $form_i
       }
 
       if ($node instanceof NodeInterface) {
-        $form['enroll_for_this_event']['#access'] = \Drupal::service('social_event.enroll')->isEnabled($node);
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.